source/guide_lychee.rst

```diff
@@ -79,18 +80,38 @@ After this, the required dependencies need to be installed:
 
 ::
 
- [isabell@stardust isabell]$ Lychee/composer install --no-dev
+ [isabell@stardust isabell]$ Lychee/composer.phar install --no-dev
  Loading composer repositories with package information
  Installing dependencies from lock file
  Package operations: 0 installs, 0 updates, 52 removals
  ...
  [isabell@stardust isabell]$
 
-Finally, setup a symbolic link for the ``public`` folder within the ``html`` folder or in the current folder if you use a custom subdomain:
+Now setup a symbolic link for the ``public`` folder within the ``html`` folder:
 
 ::
 
- [isabell@stardust isabell]$ ln -s Lychee/public html/Lychee
+ [isabell@stardust isabell]$ ln -s Lychee/public html/lychee
+
+Or in the current folder if you use a custom subdomain:
+
+::
+
+ [isabell@stardust isabell]$ ln -s Lychee/public sub.domain.org
+
+As last step, the default ``.htaccess`` file needs to be adapted for the Uberspace environment for the redirection rules to work. Open the ``.htaccess`` file in the ``Lycchee\public`` directory with you favorite editor and add the statement ``RewriteBase /``:
```

Comment:
The additional `RewriteBase` rule as well as subdomain handling is already [documented in the manual](https://manual.uberspace.de/web-documentroot.html#additional-documentroots). If I am not mistaken, there isn't any special lychee setup here.

The guides should be as short as possible and not duplicate any manual content. Please remove the additional steps.